Kafka_Consumer.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
- "examples" directory is created.
- "Kafka_Consumer.py" is renamed to be "log_consumer.py" in examples folder.